### DISCUSSION TITLE
DayZ

### DISCUSSION THREAD

**Discussion by mvaessen** [gameserverapp/Platform#27] [Suggestions]
https://community.bistudio.com/wiki/DayZ:Server_Configuration

**Comment by 9TG3** [comment]
Need this one desperately. If there's anything I can do to help, let me know.

**Comment by LimitlessDrag0n** [comment]
This one would be great, but I think, if Max will get this to run, there is a big feature GSA needs. That feature would be awesome for all games though.

An in-build Steam Workshop installer. I'm hosting a DayZ on another platform and they make installing mods, pretty easy. You only need to search for them and click install. A huge plus especially for DayZ Mod installing :D

**Reply by mvaessen** [reply]
I'm assuming this is identical to how ARK uses mods?

**Reply by LimitlessDrag0n** [reply]
On ARK you only need the ID to install the mod. If you're self-hosting a DayZ Server you need to download the wanted mods on your machine and upload the mods into the root folder. Also, there is a "key" in all mod folders of DayZ that need to be copied into the root key folder. Also, all mods need to be added to the start.bat. Some servers take "spaces" in mod folder names, some others need to be renamed with a "_" so there is no space in the name :D

Don't know how my current provider is managing this but it works.

**Reply by mvaessen** [reply]
right, that sounds pretty similar to how it works for ARK and Conan.
So GSA has the required infrastructure in place already to manage mods.

**Reply by LimitlessDrag0n** [reply]
Okay, now we only need to wait that DayZ brings a stable Linux installation :D Still in experimental :D

**Comment by LarryBambel** [comment]
I think that if Max, puts in access an interface like ARK for DayZ
It will attract a lot of people, because this game is widely played and can thus attract other people.
This will be a lot of work, but many server admins could import their servers into such a clean interface like GSA

I vote for the implementation of DAYZ

**Comment by furio85** [comment]
Have to agree.  I would def vote for DAYZ!
